include/ignition/common/EnumIface.hh

```diff
@@ -17,3 +17,6 @@
 
 #include <gz/common/EnumIface.hh>
 #include <ignition/common/config.hh>
+
+#define IGN_ENUM(name, enumType, begin, end, ...) \
+  GZ_ENUM(name, enumType, begin, end, ...)
```

Comment:
I think the `...` need to be `__VA_ARGS__`

Reply:
https://github.com/gazebosim/gz-common/pull/366/commits/4c8849120b890b333e4561cd50985e456c186b97